### ISSUE TITLE
Working basic Openwrt image for Waveshare Dual LAN board for Rpi CM4

### ISSUE THREAD

**Issue by daserra23** [geerlingguy/raspberry-pi-pcie-devices#557]
Wondering if anyone can help, I bought [this board](https://www.amazon.de/dp/B09LV2L9NM?psc=1&ref=ppx_yo2ov_dt_b_product_details) and a Raspberry pi CM4 from Amazon to build a home router but the OpenWrt images from the Waveshare site do not work, they are unpingable from the IP address's (192.168.14.1 or 192.168.1.1). 
I tried a multitude of options with and without DHCP but nothing was pingable and I was never allocated an IP.

I would also prefer an image direct from openwrt.org but I think compiling my own is beyond me.

However the stock CM4 image from openwrt.org _does_ work but the second USB RTL8153 built in adaptor is not available when I try to add another connection (for the WAN).

Ideally it would be great if anyone could send me a link to a working image with the drivers pre-installed or some method of adding them to an OpenWRT router without internet.

**Comment by daserra23** [comment]
I had to connect the pi to a normal network via eth0 with DHCP off in Openwrt using a static IP in the standard range of the normal network and setting DNS to 1.1.1.1. 

Then I could log in WITH internet and "update lists" in the software section of openwrt.

Then I could install usbutils and kmod-usb-net-rtl8152 which made the adaptor available.

Then I set it up as a "DHCP client" so that the upstream internet router (set to pass through) was able to provide internet.
